zuora.gemspec

```diff
@@ -1,36 +1,47 @@
-# -*- encoding: utf-8 -*-
-$:.push File.expand_path("../lib", __FILE__)
-require "zuora/version"
+# frozen_string_literal: true
+
+$:.push File.expand_path('../lib', __FILE__)
+require 'zuora/version'
 
 Gem::Specification.new do |s|
-  s.name        = "zuora"
+  s.name        = 'zuora'
   s.version     = Zuora::Version.to_s
+  s.authors     = ['Sage Accounting']
+  s.email       = ['sageone@sage.com']
+  s.summary     = 'Zuora - ActiveModel backed client for the Zuora API'
+  s.description = 'Zuora - Easily integrate the Zuora SOAP API using ruby objects.'
+  s.homepage    = 'https://github.com/zuorasc/zuora'
+
+  s.files       = Dir.glob('{bin,lib}/**/**/**')
+  s.bindir = 'exe'
+  s.executables = s.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  s.require_paths = ['lib']
   s.platform    = Gem::Platform::RUBY
-  s.authors     = ["Josh Martin"]
-  s.email       = ["josh.martin@wildfireapp.com"]
-  s.homepage    = "http://github.com/wildfireapp/zuora"
-  s.summary     = %q{Zuora - ActiveModel backed client for the Zuora API}
-  s.description = %q{Zuora - Easily integrate the Zuora SOAP API using ruby objects.}
-  s.files         = `git ls-files`.split("\n")
-  s.test_files    = `git ls-files -- {test,spec,features}/*`.split("\n")
-  s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
-  s.require_paths = ["lib"]
-  s.extra_rdoc_files = [ "README.md" ]
+  s.test_files  = `git ls-files -- {test,spec,features}/*`.split("\n")
+  s.extra_rdoc_files = ['README.md']
 
-  s.add_runtime_dependency('savon', ["0.9.9"])
-  s.add_runtime_dependency('activesupport', [">= 3.0.0", "< 5.0.0"])
-  s.add_runtime_dependency('activemodel', [">= 3.0.0", "< 5.0.0"])
-  s.add_runtime_dependency('httpi', ["0.9.7"])
-  s.add_runtime_dependency('libxml4r', ['~> 0.2.6'])
+  s.add_development_dependency 'akami', '<= 1.3.2'
+  s.add_development_dependency 'appraisal', '~> 0.4.1'
+  s.add_development_dependency 'artifice', '~> 0.6.0'
+  s.add_development_dependency 'bigdecimal', '< 2.0.0'
+  s.add_development_dependency 'factory_girl', '~> 2.6.4'
+  s.add_development_dependency 'ffi', '<= 1.16.3'
+  s.add_development_dependency 'guard-rspec', '~> 0.6.0'
+  s.add_development_dependency 'nokogiri', '~> 1.15.6'
+  s.add_development_dependency 'pry'
+  s.add_development_dependency 'pry-nav'
+  s.add_development_dependency 'rake', '~> 0.8.7'
+  s.add_development_dependency 'redcarpet', '~> 2.1.0'
+  s.add_development_dependency 'rspec', '~> 3.0'
+  s.add_development_dependency 'simplecov', '~> 0.6.4'
+  s.add_development_dependency 'sqlite3', '~> 1.3.0'
+  s.add_development_dependency 'test-unit'
+  s.add_development_dependency 'wasabi', '2.3.0'
```

Comment:
And wasabi should go in the Gemfile only too, as it is also a dependency of savon.

Reply:
Addressed in this [commit](https://github.com/Sage/zuora/pull/14/commits/62ad549ca14815cf6660155b1f87c6399822e385)